src/main/java/maksym/fedorenko/bookstore/controller/BookController.java

```diff
@@ -57,6 +58,7 @@ public BookDto getBookById(@PathVariable @Positive Long id) {
     )
```

Comment:
a general recommendation, add `@PreAuthorize("hasRole('USER')")` to the relevant endpoints to avoid misunderstandings in the future